Cargo.toml

```diff
@@ -129,4 +129,7 @@ wasmparser = { version = "0.110", default-features = false }
 wasmtime = { version = "11", default-features = false }
 wasmtime-wasi = { version = "11", default-features = false }
 wit-bindgen = { version = "0.9", default-features = false }
+wit-bindgen-wasmcloud-provider = { path = "./crates/wit-bindgen-wasmcloud-provider" }
 wit-component = { version = "0.12", default-features = false }
+heck = { version = "0.4.1" }
+proc-macro2 = { version = "1.0.66" }
```

Comment:
All other deps in this file are (or, at least, should be) sorted lexicographically. If you choose not to do this now - that's fine, but could you add a TODO here then? These also should not blindly enable all default features, but only the features required, just like all other dependencies in this repository

Reply:
- [ ] remove default feature usage from deps